native_client/deepspeech.cc

```diff
@@ -322,6 +372,41 @@ ModelState::infer(const float* aMfcc, unsigned int n_frames, vector<float>& logi
   for (int t = 0; t < n_frames * BATCH_SIZE * num_classes; ++t) {
     logits_output.push_back(logits_mapped(t));
   }
+#else // USE_TFLITE
+  // Feeding input_node
+  float* input_node = interpreter->typed_tensor<float>(interpreter->inputs()[0]);
```

Comment:
Are these integer input/output IDs stable? For example, do they resist model architecture changes?

Reply:
We could add checks against the names if you want: https://github.com/mozilla/tensorflow/blob/r1.12/tensorflow/contrib/lite/interpreter.h#L199-L218